Documentation/kubernetes.md

```diff
@@ -93,12 +93,20 @@ __Caveats:__ No health checking is configured because dex does its own TLS termi
 
 ## Logging into the cluster
 
-The example app can be used to log into the cluster. Choose the GitHub option and grant access to dex to view your profile.
+The `example-app` can be used to log into the cluster and get an ID Token.
+
+To build the app, you can run `make` and it will build the `example-app` binary in the root of the repo in the `bin` directory. To build the `example-app` requires at least a 1.7 version of Go. 
```

Comment:
Probably doesn't need to be a new paragraph.

Also we need to specify that `make` is also run at the root of the directory.